projects/L4T/packages/glibc/package.mk

```diff
@@ -0,0 +1,168 @@
+PKG_NAME="glibc"
+PKG_VERSION="2.27"
+PKG_LICENSE="GPL"
+PKG_SITE="http://www.gnu.org/software/libc/"
+PKG_URL="http://ftp.gnu.org/pub/gnu/glibc/$PKG_NAME-$PKG_VERSION.tar.xz"
+PKG_DEPENDS_TARGET="ccache:host autotools:host autoconf:host linux:host gcc:bootstrap"
+PKG_DEPENDS_INIT="glibc"
+PKG_SECTION="toolchain/devel"
+PKG_SHORTDESC="glibc: The GNU C library"
+PKG_LONGDESC="The Glibc package contains the main C library. This library provides the basic routines for allocating memory, searching directories, opening and closing files, reading and writing files, string handling, pattern matching, arithmetic, and so on."
+PKG_BUILD_FLAGS="-gold -lto"
+
+PKG_CONFIGURE_OPTS_TARGET="BASH_SHELL=/bin/sh \
+                           ac_cv_path_PERL=no \
+                           ac_cv_prog_MAKEINFO= \
+                           --libexecdir=/usr/lib/glibc \
+                           --cache-file=config.cache \
+                           --disable-profile \
+                           --disable-sanity-checks \
+                           --enable-add-ons \
+                           --enable-bind-now \
+                           --with-elf \
+                           --with-tls \
+                           --with-__thread \
+                           --with-binutils=${BUILD}/toolchain/bin \
+                           --with-headers=${SYSROOT_PREFIX}/usr/include \
+                           --enable-kernel=3.0.0 \
+                           --without-cvs \
+                           --without-gd \
+                           --disable-obsolete-rpc \
+                           --disable-build-nscd \
+                           --disable-nscd \
+                           --enable-lock-elision \
+                           --disable-timezone-tools"
+
+if [ "${DEBUG}" = yes ]; then
+  PKG_CONFIGURE_OPTS_TARGET="${PKG_CONFIGURE_OPTS_TARGET} --enable-debug"
+else
+  PKG_CONFIGURE_OPTS_TARGET="${PKG_CONFIGURE_OPTS_TARGET} --disable-debug"
+fi
```

Comment:
use PKG_CONFIGURE_OPTS_TARGET+=" ..."

Reply:
I am trying to not have stuff in filesystem folder if I can avoid it. Especially for something that could easily be adapted to work with other boards.